tds-dll-common/src/main/java/tds/dll/common/diagnostic/controller/AbstractStatusController.java

```diff
@@ -19,6 +20,7 @@
 import java.util.Date;
 import java.util.List;
 
+@Controller
```

Comment:
Since this class can never be instantiated are we sure this is needed??  I would imagine the implementing classes will have the annotation.